Makefile

```diff
@@ -36,6 +36,9 @@ stop: ## Stop the container
 		docker rm --force ${EXISTING_CONTAINERS};\
 	fi
 
+cycle: ## Stop, build and restart the container
+	make stop build start
```

Comment:
Suggest `stop` is made a dependency of `start`. `restart` might be a better name?

Reply:
Can add `stop` as a dependency of `start`.

Including `stop` here too makes sure that the application is stopped when the build occurs. Not sure if building while it's running might have side effects.

Reply:
Rejigged in https://github.com/libero/storybook/pull/13/commits/e788e6c7c0a486c786310c16e322865320f8a1b2 so that:
- `start` is renamed to `restart`: it runs `make stop` before, err, starting
- `cycle` renamed to `rebuild`: it runs `make stop build restart`

This does mean that `stop` is called twice during `cycle`, but that seems harmless.